assets/acquia-spec.version

```diff
@@ -0,0 +1 @@
+3035ccd40bea9225a8b2a0fa421ffd9bde7b16da
```

Comment:
This just makes it easier to track what version of the spec we're using, since it's not encoded in acquia-spec.json itself